package.json

```diff
@@ -173,5 +177,25 @@
     "@ungap/custom-elements-builtin": "^0.1.1",
     "document-register-element": "^1.12.0",
     "react": ">=0.14.0"
+  },
+  "jest": {
```

Comment:
Personally I prefer to keep configuration scoped in separate config files.
May we moved it `jest.config.js` as described here: https://jestjs.io/docs/en/configuration.html